config/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "protocol": "http",
+  "host": "152.46.18.48",
```

Comment:
Maybe this should be renamed to `raginmages.net` to keep it consistent with the instructions below.